From bd3ca6d0cfb89273cd7fb71428cc2165d82de699 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 6 Jul 2022 11:03:32 +0200
Subject: [PATCH 0163/1141] arm64: defconfig: enable support of the STM32 spi
 controller

Add compilation of CONFIG_SPI_STM32 as a module in
fragment-02-defconfig-addons

Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Change-Id: I00ed9464217391e1d09ad2270e0e301f3450acc3
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/301586
Tested-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -20,6 +20,7 @@ CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_RTC_DRV_STM32=y
 CONFIG_SERIAL_STM32=y
 CONFIG_SERIAL_STM32_CONSOLE=y
+CONFIG_SPI_STM32=m
 CONFIG_STM32_DMA3=y
 CONFIG_STMMAC_ETH=y
 CONFIG_STMMAC_PLATFORM=y
